charts/influxdb-enterprise/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 0.1.22
+version: 0.1.24
```

Comment:
Does this need a bump for 2 minor versions, I think simply `0.1.23` here is sufficient.

Reply:
Check was failing for 1.23 😭 I'll try again but I was just listening to CI 😆

Reply:
Cool it worked -- not sure why it was having issues previously. Maybe it was something not related 😅